### ISSUE TITLE
Combined g3_param_table / g3l_random_dnorm()

### ISSUE THREAD

**Issue by lentinj** [gadget-framework/gadget3#181]
We have random deviates with ``g3_param_table()`` & ``g3l_random_dnorm()``, need to expand this for a projection.

* Create a g3s_modeltime array of values
* Populate initial selection with g3_param_table
* Populate gaps with ``rnorm()``, based on ``g3l_random_dnorm()`` settings
* Use that instead of g3_param_table()

See:

* https://github.com/vbartolino/g3_GoB/blob/4e38a56e360c91ae05cbc2c32ec86d1d94b23b55/vendace_g3/gadget3/09-SRR-RND/setup_model.R#L151-L155
* https://github.com/vbartolino/g3_GoB/blob/4e38a56e360c91ae05cbc2c32ec86d1d94b23b55/vendace_g3/gadget3/09-SRR-RND/setup-random.R#L5-L14

**Comment by lentinj** [comment]
@bthe / @vbartolino for reporting of values, would it be better to have a combined array of random deviates & projected deviates, or just projected values and rely on parameter table for former? I'm assuming a combined array is more useful, but would be good to know if you think otherwise.

**Comment by bthe** [comment]
Would this mean that the values for the random effects would not appear in the parameter table?

**Comment by vbartolino** [comment]
does one option exclude the other? if possible I think it would be useful to have both and in any case intuitively a user would expect to find the parameters of the random effect in the parameter table

**Comment by lentinj** [comment]
Yes, the random effects will be in the parameter table regardless, having them in the report array won't affect that.

This is mostly a question of how to structure the array internally. If we want a report to include the random effects before projection, then I'll arrange to code so that happens by default.

**Comment by bthe** [comment]
OK, then I would think that most logical thing would be, as you suggest, to have both the estimated random effect and projected values.
